### ISSUE TITLE
Bug - variables named actor break formatting

### ISSUE THREAD

**Issue by davidelorenzitw** [nicklockwood/SwiftFormat#958]
Version used: 0.48.5

Description:
When parsing code that contain the actor word, formatting seems to break, and as a result code might not compile anymore either.

Example:
Before running SwiftFormat
```
// within my own type
init(actor: MyUserActor,
         router: MyRouter) {
        self.actor = actor
        self.router = router
```

After running SwiftFormat:
```
init(actor: MyUserActor,
         router: MyRouter) {
        self.actor = actor
        router = router // < ---- 'self.' was removed as it's coming right after the actor keyword. 
                       // As a result, code doesn't compile anymore, as 'self.' is needed here
```

**Comment by nicklockwood** [comment]
@davidelorenzitw thanks for reporting this. I'll fix it asap. In the meantime, a simple workaround in this case seems to be to reverse the order of the assignments:

```swift
init(actor: MyUserActor,
         router: MyRouter) {
        self.router = router
        self.actor = actor
}
```

(Specifically, you need to ensure that `actor` is not followed by another identifier).

**Comment by nicklockwood** [comment]
@davidelorenzitw fixed in 0.48.6
